ale/base/label_isis.py

```diff
@@ -52,12 +55,41 @@ def target_name(self):
         return self.label['IsisCube']['Instrument']['TargetName']
 
     @property
-    def starting_ephemeris_time(self):
-        if not hasattr(self, '_starting_ephemeris_time'):
-            sclock = self.label['IsisCube']['Archive']['SpacecraftClockStartCount']
-            self._starting_ephemeris_time = spice.scs2e(self.spacecraft_id, sclock).value
-        return self._starting_ephemeris_time
+    def spacecraft_clock_start_count(self):
+        """
+        The spacecraft clock start count, frequently used to determine the start time
+        of the image.
+
+        Returns
+        -------
+        : str
+          Spacecraft clock start count
+        """
+        try:
+            start_count = self.label['IsisCube']['Instrument']['SpacecraftClockStartCount']
+        except: 
+            start_count = self.label['IsisCube']['Archive']['SpacecraftClockStartCount']
+
+        return start_count
 
     @property
     def _exposure_duration(self):
```

Comment:
The `_` should be removed from this property because it's part of the public interface.